cityofaustin/atd-vz-data
# Add delete permissions for vz-admin role for changes table

## Associated issues

This PR intends to bring the changes that will close https://github.com/cityofaustin/atd-data-tech/issues/16338 into master.

## Production patch PR

#1405 

## Lineage

This branch came off of `production`, so we'll be bringing any commits we have in `production` but not into `master` back into `master`. I didn't expect to see 16 come back 😵‍💫 , but you can see, the only material change is the one file we're targeting. 

## Testing

* Spin up the DB
* Apply metadata from this branch
* Confirm `vz-admin` has delete permissions for `atd_txdot_changes` in the `hasura console`. 

---
#### Ship list
